oauth2_http/java/com/google/auth/oauth2/IamUtils.java

```diff
@@ -125,4 +128,67 @@ private static String getSignature(String serviceAccountEmail, Credentials crede
     GenericData responseData = response.parseAs(GenericData.class);
     return OAuth2Utils.validateString(responseData, "signedBlob", PARSE_ERROR_SIGNATURE);
   }
-}
+
+  /**
+   * Returns an IdToken issued to the serviceAccount with a specified
+   * targetAudience.
+   *
+   * @param serviceAccountEmail the email address for the service account to get
```

Comment:
probably ID token if it's not code since Id and ID are different things

Reply:
ok